subsys/bluetooth/controller/Kconfig

```diff
@@ -164,6 +164,7 @@ config BT_CTLR_TX_PWR_PLUS_5
 
 config BT_CTLR_TX_PWR_PLUS_4
 	bool "+4 dBm"
+	depends on SOC_SERIES_NRF51X || SOC_COMPATIBLE_NRF52X
```

Comment:
Can we make something like TX_PWR_PLUS_4_SUPPORT, and have nrf521x and nrf52x select them instead?

Reply:
checks for each CPU is hard to maintain

Reply:
Yes, I also agree on that.

Reply:
Mind however, @joerchan @kl-cruz that this needs to be done for all power levels that are SoC specific. Could we avoid to do this in the current PR? IF so, I can open an issue to introduce thes "capabilities" to the nRF SoCs

Reply:
> Could we avoid to do this in the current PR?

Yes, absolutely

Reply:
Sure